src/Mobs/Wolf.cpp

```diff
@@ -18,7 +18,7 @@ cWolf::cWolf(void) :
 	m_IsBegging(false),
 	m_IsAngry(false),
 	m_OwnerName(""),
-	m_CollarColor(14)
+	m_CollarColor(E_META_DYE_ORANGE)
```

Comment:
I seem to recall doing that. If it was me, either I had no idea what I was doing, or it was because it worked in whatever MC version the changes were implemented in.


On 1 Dec 2015, at 14:22, "Gargaj" <notifications@github.com> wrote:

> In src/Mobs/Wolf.cpp:
> 
> > @@ -18,7 +18,7 @@ cWolf::cWolf(void) :
> >  	m_IsBegging(false),
> >  	m_IsAngry(false),
> >  	m_OwnerName(""),
> > -	m_CollarColor(14)
> > +	m_CollarColor(E_META_DYE_ORANGE)
> Either way the other change (the 15-x one) is more important, and it'd be nice if someone would confirm why it was like that.
> 
> —
> Reply to this email directly or view it on GitHub.
>